modules/serverless-sinkbinding-odc.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// serverless/knative_eventing/serverless-sinkbinding.adoc
+
+[id="serverless-sinkbinding-odc_{context}"]
+= Create a SinkBinding using the Developer perspective
+
+This guide describes the steps required to create, manage, and delete a sample SinkBinding instance by using **Developer** perspective in the {product-title} web console.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have switched to the *Developer* perspective in the web console.
+
+.Procedure
+
+. Create the `event-display` sample Service to use as an event sink.
+.. Switch to the `knative-serving` **Project** in the dropdown menu.
+.. Navigate to **+Add** -> **Container Image**.
++
+image::serverless-odc-sinkbinding-1.png[Add a sample Service event sink as a Container Image]
++
+.. For the **Image** configuration, select **Image name from external registry**, then enter the sample URL:
++
+----
+quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
+----
+// TODO: Needs an update to latest sample image for 1.8.0?
++
+The **General** fields, **Application** and **Name**, will be filled automatically once the sample image is provided.
+.. For **Resources**, select **Knative Service**.
+.. Click **Create**.
+. Create a SinkBinding.
+.. Switch to the `knative-serving` **Project** in the dropdown menu.
+.. Navigate to **+Add** -> **Event Source**.
++
+image::serverless-sinkbinding-odc-2.png[Configuring SinkBinding]
++
+.. Select the **SinkBinding** icon.
+.. In the form, enter the following configuration:
+... For **apiVersion** enter `v1`.
+... For **Kind**, enter `SinkBinding`.
++
+image::serverless-sinkbinding-odc-3.png[Cofiguring a sink]
++
+... For **Sink** -> **Knative Service**, select your Service from the dropdown menu.
++
+The **General** fields, **Application** and **Name**, will be filled automatically once the sample image is provided.
++
+.. Click **Create**.
+
+.Verification steps
+
+You can verify that the Kubernetes events were sent to the Knative event sink by looking at the message dumper function logs.
```

Comment:
Unfortunately, this will not work. Previously we had here a weak verification, just by looking at how ODC renders those resources. Now, you've copied verification steps from other ways of configuring sinkbinding. This approach has couple of pitfalls:

1. We don't setup here any event producer. That means that user will not see any event on _event-display_ logs. We could fix that, by deploying some event producer like cronjob or pingsource. Best will be to deploy them using ODC only.
2. We use CLI tools to verify ODC operation. User may not have CLI configured. That's why _"ODC way"_ may have been chosen to follow. We could fix that by rewriting this CLI based verification to one that uses ODC UI to check the same outcome.
3. I'd have leave previous paragraph of how ODC should render sinkbinding being connected to ksvc, just as first step of simple verification.

Reply:
User can click om `Knative Service` in topology and on sidebar , it'll show Pods with view logs option, user can click on `view logs` and check the events in logs similar to what's being done via CLI. @cardil Thoughts?